### PR TITLE
Add modelState error when sending whitespace

### LINKED ISSUE #251: Error 404 after typing white char as an aswer
**To Reproduce**
1. Find slot for which answer wasn't taken
2. In answer field type only space and nothing else
3. Press send

**Current behavior**
image

**Expected behavior**
For sure error 404 should not be visible. Question if alone space should be treated as answer, or should be information that this field should be filled in.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by szrtGreg**
## Description
Error 404 after typing white char as an aswer

## Where should the reviewer start?
Please check issue to reproduce error. 
My changes were only in TestController - Index(int testNumber, string answer = "") method

## Motivation and context
closes #251 

## How has this been tested?
I tested manually. Added question, and sends empty answer.

## Screenshots (if appropriate)
expected behavoiur:
obraz

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs [hunk 1]
```diff
@@ -38,13 +38,14 @@ public ActionResult Index(int testNumber)
         [HttpPost]
         public ActionResult Index(int testNumber, string answer = "")
         {
-            if (answer == null)
+            var test = this._testService.GetTestByNumber(testNumber);
+
+            if (string.IsNullOrWhiteSpace(answer))
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs:43]: please first check the _answer_, then get _test_

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs [hunk 2]
```diff
@@ -38,13 +38,14 @@ public ActionResult Index(int testNumber)
...
             {
-                throw new ArgumentNullException(nameof(answer));
+                this.ModelState.AddModelError("Answers", "Podaj odpowiedź!");
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs:45]: please use resx to store error message

**Comment by WTobor** [comment]
@szrtGreg after merge please ensure that #357 handle this scenario

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs (hunk 2 above)
Comment (szrtGreg) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/TestController.cs]: done

**Review by WTobor — APPROVED** [review]

**Pull request merged**